app/static/ie-status.json

```diff
@@ -3847,6 +3847,23 @@
     "id": 6248386202173440
   },
   {
+    "name": "Web Bluetooth",
+    "category": "Device",
+    "link": "https://webbluetoothcg.github.io/web-bluetooth/",
+    "summary": "An API for communicating with Bluetooth Low Energy devices as a GATT Client.",
+    "standardStatus": "Editor's draft",
+    "ieStatus": {
+      "text": "Not currently planned",
```

Comment:
You can put Under Consideration for this.  Also, add a field under ieStatus like so:

    "priority": "low",